src/main/java/com/microfocus/octane/plugins/configuration/ConfigurationUtil.java

```diff
@@ -255,5 +262,38 @@ public static WorkspaceConfiguration validateRequiredAndConvertToInternal(Worksp
 
         return wc;
     }
+
+    private static void validateJiraProjectKey(WorkspaceConfigurationOutgoing wco) {
+        List<String> projectKeys = ComponentAccessor.getProjectManager().getProjectObjects()
+                .stream()
+                .map(Project::getKey)
+                .collect(Collectors.toList());
+
+        if (wco.getJiraProjects().stream().anyMatch(e -> !projectKeys.contains(e.trim()))) {
```

Comment:
collections have a containsAll() method

Reply:
@atothhpe interesting idea, replaced in some places